client/pages/Leagues/Leagues.tsx

```diff
@@ -25,15 +27,24 @@ const Leagues = () => {
   const [searchQuery, setSearchQuery] = useState("");
   const [debouncedQuery, setDebouncedQuery] = useState("");
 
+  const { getAccessTokenSilently } = useAuth0();
+  const [groupId, setGroupId] = useState(null);
```

Comment:
I  believe here we do need to set a type because in line 44, it will error if it sets anything other than null.

```suggestion
  const [groupId, setGroupId] = useState<number | null>(null);
```

Reply:
Done!